ambry-store/src/main/java/com.github.ambry.store/BlobStoreCompactor.java

```diff
@@ -718,7 +716,7 @@ private boolean copyRecords(LogSegment logSegmentToCopy, List<IndexEntry> srcInd
           fileChannel.position(srcValue.getOffset().getOffset());
           Offset endOffsetOfLastMessage = tgtLog.getEndOffset();
           // call into diskIOScheduler to make sure we can proceed (assuming it won't be 0).
-          diskIOScheduler.getSlice(LOG_SEGMENT_COPY_JOB_NAME, LOG_SEGMENT_COPY_JOB_NAME, writtenLastTime);
+          diskIOScheduler.getSlice(DiskManager.CLEANUP_OPS_JOB_NAME, DiskManager.CLEANUP_OPS_JOB_NAME, writtenLastTime);
```

Comment:
orthogonal question. Why we need both jobId and jobType? I don't see any usage of jobId in DiskIOScheduler.